packages/uploadthing/src/types.ts

```diff
@@ -25,11 +25,21 @@ type UploadedFile = {
   size: number;
 };
 
-type AllowedFiles = "image" | "video" | "audio" | "blob";
+export type AllowedFileType = "image" | "video" | "audio" | "blob";
 
 export type SizeUnit = "B" | "KB" | "MB" | "GB";
-type PowOf2 = 1 | 2 | 4 | 8 | 16 | 32 | 64 | 128 | 256 | 512 | 1024;
-export type FileSize = `${PowOf2}${SizeUnit}`;
+export type FileSize = `${number}${SizeUnit}`;
```

Comment:
I think a generic `number` here kills intellisense and autocompletion

Reply:
:( cc @markflorkowski